### PR TITLE
add term hyperinsulinemic hypoglycemia with polycystic kidney disease

### LINKED ISSUE #7457: Hyperinsulinemic Hypoglycemia with Polycystic Kidney Disease
### Label

hyperinsulinemic hypoglycemia with polycystic kidney disease

### Synonyms

n/a

### Synonym type

None

### Definition

Hyperinsulinemic Hypoglycemia with Polycystic Kidney Disease. The HPO terms are: HP:0000825, HP:0000113.

### Parent term

MONDO:0005803 hyperinsulinemic hypoglycemia

### Children term(s)

N/A

### ORCID Identifier

_No response_

### Website URL

https://clinicalgenome.org/affiliation/40066/

### Additional comments

_No response_

### PR DESCRIPTION
**Pull request by twhetzel**
closes #7457

### CODE REVIEW

File: src/ontology/mondo-edit.obo
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:546783]: the ClinGen Working Group should be added to the dbxref for the definition.

**Review by sabrinatoro — CHANGES_REQUESTED** [review]
Please see comments for the requested changes.
The main requested change is the definition which would be re-written as it is currently confusing and misleading. I suggest that you share the definition with the person who requested the term to make sure that they agree with it.

File: src/ontology/mondo-edit.obo
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:546783]: I updated the definition and added the source.

File: src/ontology/mondo-edit.obo
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:546785]: done

**Comment by sabrinatoro** [comment]
@twhetzel I am still suspicious about the definition and about any "autosomal recessive disorder by this specific variant" to be "hyperinsulinemic hypoglycemia with polycystic kidney disease". On the other hand, does it mean that there is absolutely no other cause for this disease? Shouldn't the definition mention the fact that it is a hyperinsulinemic hypoglycemia for example.

Definitions should be discussed and approved by the requestor before we can approve/add a new term to master (once it is in the ontology, it is official). So please get the approval by the requestor first.

**Comment by twhetzel** [comment]
@sabrinatoro I asked in the review comment that you review the updated definition before I post in the issue ticket. I gather you have done this so I've now asked the requestor for feedback. Based on the limited information I can find about this disease, this is the best I can suggest.

**Comment by twhetzel** [comment]
This PR is on hold until the definition is [verified](https://github.com/monarch-initiative/mondo/issues/7457#issuecomment-2102970216) by the user who submitted the term.

File: src/ontology/mondo-edit.obo
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:546783]: The user [agreed](https://github.com/monarch-initiative/mondo/issues/7457#issuecomment-2116152998) to the definition.

**Comment by twhetzel** [comment]
This is ready for review.

**Review by sabrinatoro — APPROVED** [review]
looks good. Approving and merging.

**Pull request merged**
